astromodels/core/parameter.py

```diff
@@ -791,11 +793,11 @@ def get_callbacks(self):
         """
         return self._callbacks
 
-    def empty_callbacks(self):
+    def empty_callbacks(self) -> None:
         """Remove all callbacks for this parameter"""
         self._callbacks = []
 
-    def duplicate(self):
+    def duplicate(self) -> "Parameter":
```

Comment:
Ah, because the typing can't handle the fact that you are declaring a Parameter return while creating the class Parameter. This is how you get around that.

Reply:
Ah, I see. Thanks!